public/global.css

```diff
@@ -80,4 +80,10 @@ content {
 	padding: 0%;
 	overflow: hidden;
 	background-color: white;
+}
+
+img {
```

Comment:
We didn’t focus much on styling patterns. If I did, I would want to make this a little less global, target a specific class or id, or move it into a component so that the scope only applies to exactly what it is meant to apply to. 

I know that the scope of the UI has narrowed a lot, so I don’t think it is worth changing css styles at this point.